### PR TITLE
[plugin.video.cbc] 4.0.1

### PR DESCRIPTION
**Pull request by micahg**
### Description
- fix auth problems hopefully
- play videos as videos
- clean up handling of live streams
- add duration
- fix http 500 status as well
- fix watched state

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.cbc/resources/lib/utils.py [hunk 1]
```diff
@@ -33,6 +34,7 @@ def loadCookies():
     try:
         with open(getCookieFile(), 'r') as f:
             cookies = pickle.load(f)
+            f.close()
```
Comment (enen92) [inline plugin.video.cbc/resources/lib/utils.py]: not needed if you use the with statement

File: plugin.video.cbc/resources/lib/utils.py [hunk 2]
```diff
@@ -57,6 +59,7 @@ def getAuthorizationFile():
 def saveAuthorization(authorization):
     with open(getAuthorizationFile(), 'w') as f:
         pickle.dump(authorization, f)
+        f.close()
```
Comment (enen92) [inline plugin.video.cbc/resources/lib/utils.py]: same

File: plugin.video.cbc/resources/lib/utils.py [hunk 3]
```diff
@@ -67,9 +70,10 @@ def loadAuthorization():
     try:
         with open(getAuthorizationFile(), 'r') as f:
             authorization = pickle.load(f)
+            f.close()
```
Comment (enen92) [inline plugin.video.cbc/resources/lib/utils.py]: same

**Comment by micahg** [comment]
Is it ok if I get those on the next revision?

**Comment by enen92** [comment]
sure :)

**Pull request merged**